stdlib/printf.mli

```diff
@@ -147,6 +147,12 @@ val ifprintf : 'b -> ('a, 'b, 'c, unit) format4 -> 'a
     @since 3.10.0
 *)
 
+val ibprintf : Buffer.t -> ('a, Buffer.t, 'c, unit) format4 -> 'a
```

Comment:
It is better to use the `format` type (and not `format4`) to get exactly the same type as `bprintf`.